build.gradle

```diff
@@ -135,7 +135,7 @@ dependencies {
     implementation 'com.blackducksoftware.bdio:bdio-protobuf:3.2.6'
     implementation "com.synopsys.integration:blackduck-common:${blackDuckCommonVersion}"
     implementation 'com.synopsys:method-analyzer-core:0.2.0'
-    implementation "${locatorGroup}:${locatorModule}:1.0.18"
+    implementation "${locatorGroup}:${locatorModule}:1.0.20"
```

Comment:
Is this a merge conflict?

Reply:
Shouldn't be. The 9.1.z branch is at 1.0.20 as well. (I had to make the change locally to build).

Reply:
Was this branch checked out from master?

Reply:
It started out from master, before 9.1.z was branched.

Reply:
Thats why it didn't get picked up in your branch, but seems fine I guess